Add `forceAfterTimeout` option (#45)
sindresorhus/fkill: readme.md

Force kill the process.

##### forceAfterTimeout

Type: `number`\
Default: `undefined`

> If not force killing, wait up to `forceAfterTimeout` number of milliseconds for process(es) to exit, then force kill the ones which didn't.

Reviewer: ```suggestion
Wait up to `forceAfterTimeout` number of milliseconds for processes to exit, then force kill the ones which didn't.

This option does nothing if the `force` option is `true`.
```
